scripts/kube_bump.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Please move this under `packages/@aws-cdk/aws-eks/scripts`

Reply:
Done